src/scenarios/mauiandroidpodcast/pre.py

```diff
@@ -0,0 +1,45 @@
+'''
+pre-command
+'''
+import requests
+import subprocess
+from mauishared.mauisharedpython import RemoveAABFiles
+from performance.logger import setup_loggers, getLogger
+from shared.precommands import PreCommands
+from shared.versionmanager import versionswritejson, GetVersionFromDllPowershell
+from shared import const
+
+setup_loggers(True)
+precommands = PreCommands()
+target_framework_wo_platform = precommands.framework.split('-')[0]
```

Comment:
Could this be, maybe, extracted and reused? Just in case we ever need to change the logic.